docs/release/releases/release-1.6.md

```diff
@@ -0,0 +1,35 @@
+# Cluster API v1.6
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.6` release cycle.
+
+| **What**                                             | **Who**      | **When**                   | **Week** |
+|------------------------------------------------------|--------------|----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 31st July 2023      | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 4th August 2023      | week 1   |
+| Team finalized                                       | Release Lead | Friday 4th August 2023      | week 1   |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 29th August 2023       | week 5   |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 26th September 2023      | week 9   |
+| v1.6.0-beta.0 released                               | Release Lead | Tuesday 24th October 2023     | week 13  |
+| Communicate beta to providers                        | Comms Manager| Tuesday 24th October 2023     | week 13  |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 24th October 2023     | week 13  |
+| v1.6.0-beta.x released                               | Release Lead | Tuesday 31st October 2023      | week 15  |
+| KubeCon idle week | Release Team | Monday 6th November to Thursday 9th November 2023 | week 15 |
+| release-1.6 branch created (**Begin [Code Freeze]**) | Release Lead | Tuesday 14th November 2023     | week 16  |
+| v1.6.0-rc.0 released                                 | Release Lead | Tuesday 14th November 2023     | week 16  |
+| release-1.6 jobs created                             | CI Manager   | Tuesday 14th November 2023     | week 16  |
+| v1.6.0-rc.x released                                 | Release Lead | Tuesday 21th November 2023     | week 17  |
+| **v1.6.0 released**                                  | Release Lead | Tuesday 28st November 2023     | week 18  |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 28st November 2023     | week 18  |
+| Organize release retrospective                       | Release Lead | TBC                        | week 17  |
```

Comment:
```suggestion
| Organize release retrospective                       | Release Lead | TBC                        | week 18  |
```
nit, otherwise lgtm

Reply:
updated